rest/src/main/java/org/jboss/pnc/rest/endpoints/BuildEndpointImpl.java

```diff
@@ -83,14 +83,8 @@ public class BuildEndpointImpl implements BuildEndpoint {
      * Param 1: build-id
      * <p>
      * Param 2: build-log/alignment-log
-     * <p>
-     * Param 3: lines per batch
-     * <p>
-     * Param 4: rate of batches (in ms)
-     * <p>
-     * Param 5: line format (PLAIN/LEVEL) {@see org.jboss.pnc.api.bifrost.enums.Format} for options
      */
-    private static final String BIFROST_LOGS_ENDPOINT = "/text?matchFilters=mdc.processContext:build-{0},loggerName:org.jboss.pnc._userlog_.{1}&batchSize={2}&batchDelay={3}&format={4}&direction=ASC";
+    private static final String BIFROST_LOGS_ENDPOINT = "/final-log/{0}/{1}";
```

Comment:
shouldn't it still be `build-{0}`?

Reply:
not from what I used like: `http://bifrost.example.com/final-log/A66P4Z2Q2NQAA/build-log`
cc @thescouser89 is this correct?

Reply:
Yup correct! This is what I've been using on devel.

I think if all cases processContext = buildId